keyboards/preonic/keymaps/trudkin/mykb.json

```diff
@@ -0,0 +1 @@
__________DVORAK_L2_________________", "_________________DVORAK_R2_________________", "_________________DVORAK_L3_________________", "_________________DVORAK_R3_________________", "SPACE_CLUSTER_2x2"], ["___________________BLANK___________________", "___________________BLANK___________________", "___________________BLANK___________________", "___________________BLANK___________________", "___________________BLANK___________________", "___________________BLANK___________________", "___________________BLANK___________________", "___________________BLANK___________________", "SPACE_CLUSTER_1x2"], ["MY_ESC", "_________________FUNC_LEFT_________________", "_________________FUNC_RIGHT________________", "KC_BSPC", "KC_INS", "_________________LOWER_L1__________________", "_________________LOWER_R1__________________", "KC_EQL", "KC_DEL", "_________________LOWER_L2__________________", "_________________LOWER_R2__________________", "KC_PLUS", "KC_TRNS", "_________________LOWER_L3__________________", "_________________LOWER_R3__________________", "KC_NO", "KC_TRNS", "NUMPAD", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_HOME", "KC_PGDN", "KC_END"], ["KC_ESC", "________________NUMBER_LEFT________________", "________________NUMBER_RIGHT_______________", "KC_BSPC", "KC_INS", "_________________RAISE_L1__________________", "_________________RAISE_R1__________________", "KC_NO", "KC_DEL", "_________________RAISE_L2__________________", "_________________RAISE_R2__________________", "KC_BSLS", "KC_TRNS", "_________________RAISE_L3__________________", "_________________RAISE_R3__________________", "RGB_TOG", "KC_TRNS", "NUMPAD", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_MSTP", "KC_VOLD", "KC_MPLY"], ["KC_ESC", "QWERTY", "COLEMAK", "DVORAK", "KC_NO", "KC_PSCR", "KC_NLCK", "RGB_SPD", "RGB_MOD", "RGB_RMOD", "RGB_TOG", "KC_MAKE", "RESET", "DEBUG", "KC_TRNS", "KC_TRNS", "TERM_ON", "TERM_OFF", "RGB_VAI", "RGB_VAD", "RGB_M_R", "RGB_M_X", "RGB_M_B", "KC_TRNS", "KC_TRNS", "MU_MOD", "AU_ON", "AU_OFF", "AG_NORM", "AG_SWAP", "RGB_HUI", "RGB_HUD", "RGB_M_K", "RGB_M_SW", "RGB_M_SN", "KC_TRNS", "MUV_DE", "MUV_IN", "MU_ON", "MU_OFF", "MI_ON", "MI_OFF", "RGB_SAI", "RGB_SAD", "RGB_M_T", "RGB_M_X", "RGB_M_G", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "_________________NUMPAD_1__________________", "_________________NUMPAD_1__________________", "KC_TRNS", "KC_TRNS", "_________________NUMPAD_2__________________", "_________________NUMPAD_2__________________", "KC_TRNS", "KC_TRNS", "_________________NUMPAD_3__________________", "_________________NUMPAD_3__________________", "KC_TRNS", "KC_TRNS", "KC_TRNS", "_______NUMPAD_4________", "KC_TRNS", "KC_TRNS", "_______NUMPAD_4________", "KC_TRNS", "KC_TRNS", "KC_NO", "NUMPAD", "KC_NO", "KC_NO", "KC_TRNS", "KC_NO", "KC_NO", "KC_TRNS", "KC_NO", "KC_NO", "KC_NO", "KC_NO"]]}
```

Comment:
This file should be removed; it's not going to work with any of QMK's tooling.

`git rm --cached -- keyboards/preonic/keymaps/trudkin/mykb.json` to remove it from Git while leaving it present on your system. Omit the `--cached` argument if you want to remove it from Git *and* your system.

Reply:
changes  made

Reply:
I'm not seeing this change on the pull request. You probably need to merge your GitHub fork's `master` branch with your local (on your computer) `master` branch, then run:

```
git commit -m "Delete keyboards/preonic/keymaps/trudkin/mykb.json"
git push
```

Reply:
This should be correct